packages/cyberstorm/src/components/componentStyles/CustomSelectItem.module.css

```diff
@@ -0,0 +1,50 @@
+.root {
+  display: flex;
+  flex-direction: row;
+
+  gap: var(--gap-m);
+  min-width: 10rem;
+  padding: var(--padding-3xs) var(--padding-xs);
```

Comment:
I'd change the vertical padding to `--padding-2xs` to be more in line with the designs.